starter-code/apple-pie-starter-code/css/main.css

```diff
@@ -0,0 +1,52 @@
+header {
+  background-image: url("../images/apple-pie.jpg");
+  background-size: cover;
+  height: 650px;
+}
+
+body {
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+  font-weight: lighter;
+  font-size: 20px;
```

Comment:
if you wanted to remove the white space from around the header all you need to is add a 'margin:0 'to the body